### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/laravel-validation-rules/country-codes/pull/13/commits/fc772be65e0a60f0fe744ac55d30905def7c2f1f?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/country-codes.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "laravel-validation-rules/country-codes": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        php: [7.3, 7.4, 8.0]
+        php: [7.3, 7.4, 8.0, 8.0]
```
Comment (clarkeash) [inline .github/workflows/tests.yml:20]: PHP 8.0 should not be added twice

Comment (jasonmccreary) [reply .github/workflows/tests.yml:20]: Yeah, little glitch there. We'll get that fixed up for the future.

**Pull request merged**